### ISSUE TITLE
OIDC login fails - Boolean variables are not accessed correctly

### ISSUE THREAD

**Issue by nodomain** [open-webui/open-webui#6384]
# Bug Report

## Installation Method

Docker (AWS ECS)

## Environment

- **Open WebUI Version:** v0.3.33

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [x] I have included the Docker container logs.
- [x] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

Login via OIDC EntraID works

## Actual Behavior:

Internal Server Error happens.

## Description

**Bug Summary:**
Boolean variables are not accessed correctly

## Reproduction Details

**Steps to Reproduce:**
Set the needed OIDC variables in the ECS Task Definition e.g.
```json
    { name = "ENABLE_OAUTH_SIGNUP", value = "true" },           # Boolean value as string
    { name = "OAUTH_MERGE_ACCOUNTS_BY_EMAIL", value = "true" }, # Boolean value as string
```

## Logs and Screenshots

```text
...
 File "/usr/local/lib/python3.11/site-packages/fastapi/routing.py", line 278, in app
    raw_response = await run_endpoint_function(
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/fastapi/routing.py", line 191, in run_endpoint_function
    return await dependant.call(**values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/backend/open_webui/main.py", line 2372, in oauth_callback
    return await oauth_manager.handle_callback(provider, request, response)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/backend/open_webui/utils/oauth.py", line 165, in handle_callback
    if auth_manager_config.OAUTH_MERGE_ACCOUNTS_BY_EMAIL.value:
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
...

```

## Additional Information

The boolean values seem to be accessible without ".value" and hence the call fails.

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by nodomain** [comment]
Console logs:

 
        
       GET https://xxx/oauth/microsoft/callback?code=0xxxxtvLFSTk&session_state=bff35b30-41f6-4247-a598-b456dcfd4120 500 (Internal Server Error)

**Comment by tjbck** [comment]
@Cyb4Black

**Comment by tjbck** [comment]
PR Welcome!

**Comment by nodomain** [comment]
Already getting familiar with it ;)

**Comment by brantje** [comment]
If i remember correctly,on dev branch before the release, oauth login was broken, even with the booleans set.
Once a PR is ready i'll gladly test it. Since it seems @nodomain is already working on it :).

**Comment by nodomain** [comment]
I think i fixed it, at least works for me... 
Since I am totally new to the codebase, I appreciate feedback...

**Comment by Cyb4Black** [comment]
Yes, the .value calls both need to be removed. They were actually already present before my larger OAuth changes. I'm pretty sure I removed them while coding, but it seems that some copy-pasting from the old function into my rework brought them back 😅.

So, #6392 is correctly fixing this.

**Comment by tjbck** [comment]
Seems to have been resolved, 0.3.34 will be released shortly! Thanks everyone!
